multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -81,3 +81,8 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 
 # Hyperkube image repository to use.
 # hyperkubeImageRepo: quay.io/coreos/hyperkube
+
+# AWS Tags for cloudformation stack (formatted as a list of key-value pairs)
+#tags:
+#- Name: "Kubernetes" 
+#- Environment: "Production"
```

Comment:
Is there any reason this is a list of key/value pair objects, and not just an object?